source/topics/paas/how-to-setup-application-monitoring.md

```diff
@@ -0,0 +1,9 @@
+Aptible recommends New Relic for application monitoring, and now supports it at the [host-level](https://github.com/aptible/support/issues/115) as well.
+
+(_Note: When running the New Relic Server Monitoring product in combination with the Application Monitoring product, customers are only charged per host instead of per app process._)
```

Comment:
This note seems out of left field for this context. I'd either fully explain the benefits of Server Monitoring and how it reduces costs, or omit this side note.